experiments/20160805-mutagenesis/20160831-mutagenesis.md

```diff
@@ -86,5 +86,21 @@ Gel Parameters
 - 150 volts, 20 min
 
 Gel Order
+
 | Ladder | P1 | P2 | PW | PT | A1 | A2 | Aw | AT | Ladder |
 |--------|----|----|----|----|----|----|----|----|--------|
+
+![mutagenesis_gel](./20160901-mutagenesis-pcr-gel.jpg)
+
+## Analysis
+
+Only PCR with P1 (band at 2.7kb) appears successful.
+
+**Expected Sizes**
+
+| Fragment        | expected size |
+|---------------|---------------|
+| 1: PB2 Part 1  | 1600          |
```

Comment:
For ease of reading here, could you change "1:" to "A1:", "2:" to "A2:", "3:" to "P1" and "4:" to "P2"?